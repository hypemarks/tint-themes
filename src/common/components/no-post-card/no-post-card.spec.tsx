import { render } from '@testing-library/react';
import React from 'react';
import NoPostCard from './no-post-card';

describe('Test <NoPostCard /> Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(<NoPostCard />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
