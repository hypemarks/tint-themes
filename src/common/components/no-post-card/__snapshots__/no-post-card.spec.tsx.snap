// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <NoPostCard /> Component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="no-post-card"
  >
    There are no posts here to view yet.
  </div>
</DocumentFragment>
`;
